json-rpc/types/src/request.rs

```diff
@@ -319,6 +319,8 @@ pub struct GetAccountStateWithProofParams {
 pub struct GetTransactionsWithProofsParams {
     pub start_version: u64,
     pub limit: u64,
+    // TODO(philiphayes): do we need default=false for backwards compatibility?
+    pub include_events: bool,
```

Comment:
I think this field should be optional, not because BC, but because it is an option.

Reply:
counter example is that the corresponding non-verifying api doesn't have this param as optional

Reply:
I think non-verifying API made mistake to enforce it as required parameter. And probably because at the time we create the API, there is no optional for creating optional parameter.

Reply:
I still don't have strong feelings but if we do want to default it to false then that should be a simple annotation.